fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/FedoraResourceImpl.java

```diff
@@ -493,6 +494,8 @@ public void replaceProperties(final IdentifierConverter<Resource, FedoraResource
         try {
             new RdfRemover(idTranslator, getSession(), replacementStream
                     .withThisContext(differencer)).consume();
+        } catch (final ConstraintViolationException e) {
```

Comment:
You mean like we are doing directly below this with the MalformedRdfException? 
I was a little concerned about how we are reporting if we violate multiple constraints. In which case maybe our constraints should be a single file and the exception message would have to identify which constraint? Not sure...